src/Wrapper.js

```diff
@@ -200,14 +213,16 @@ class Wrapper extends React.PureComponent {
     this.setState({ appLoading: false })
   }
 
-  // params need to be a string
-  handleParamsRequest = params => {
-    this.openApp(this.props.locator.instanceId, { params })
+  handleAppMessage = ({ data: { name, value } }) => {
```

Comment:
😅 I didn't even realize before that we removed this function and we were always passing down `undefined`